envoy/service/auth/v2/external_auth.proto

```diff
@@ -26,4 +26,22 @@ message CheckRequest {
 message CheckResponse {
   // Status `OK` allows the request. Any other status indicates the request should be denied.
   google.rpc.Status status = 1;
+
+  // An optional message which contains the attributes of an HTTP response object. This message is
+  // particularly useful when the authorization service needs to send specific responses to the
+  // downstream client or to modify the request headers before dispatching them to the upstream.
+  // E.g., when the authorization status is not OK, this message can be used by the authorization
```

Comment:
Could you clarify the behavior when the response does not include a `HttpResponse` at all for both `OK` and `not OK` cases. I assume that there is no expectation for a `HttpResponse` to be there and in that case the behavior will not change from what is there today... right?

Reply:
That's right if no extra attributes are passed from a gRPC authz service to the filter, the behavior should continue the same.